eventstore/random_contacts.py

```diff
@@ -0,0 +1,109 @@
+import requests
+from urllib.parse import urljoin
+from django.conf import settings
+from ndoh_hub.celery import app
+from celery.exceptions import SoftTimeLimitExceeded
+from requests.exceptions import RequestException
+from temba_client.exceptions import TembaHttpError
+import datetime
+from ndoh_hub.utils import rapidpro, get_today
+
+
+if settings.EXTERNAL_REGISTRATIONS_V2:
+    if settings.TURN_URL and settings.TURN_TOKEN:
+        turn_api = settings.TURN_URL + "/v1/contacts/{}/messages"
+        turn_header = {
+            "Authorization": "Bearer {}".format(settings.TURN_TOKEN),
+            "Accept": "application/vnd.v1+json",
+        }
+
+    if settings.RAPIDPRO_URL and settings.RAPIDPRO_TOKEN:
```

Comment:
Don't think we need to check if settings.RAPIDPRO_TOKEN exists, we're not using it